tests/test_trait_pipelines.py

```diff
@@ -67,7 +123,12 @@ def test_dicot_pipeline(
     canola_lateral_slp,
     soy_primary_slp,
     soy_lateral_slp,
+    canola_traits_csv,
```

Comment:
add a docstring explaining this test. particularly mentioning that it compares the computed traits using `DicotPipeline` with a fixture, with the manually computed traits from function chaining